lmdeploy/pytorch/models/internvl_patch.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Optional
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+from transformers.configuration_utils import PretrainedConfig
+
+
+class InternVisionEmbeddings(nn.Module):
```

Comment:
what is this for? What's the diference between it and `InternVisionEmbeddings` in internvl.py?

Reply:
internvl_patch.py is used to support `Mono-InternVL`.
https://huggingface.co/OpenGVLab/Mono-InternVL-2B/blob/main/modeling_intern_patch.py